src/include/migraphx/op/argmax.hpp

```diff
@@ -56,7 +56,7 @@ struct argmax
 
     shape normalize_compute_shape(std::vector<shape> inputs) const
     {
-        check_shapes{inputs, *this}.has(1);
+        check_shapes{inputs, *this, true}.has(1);
```

Comment:
This is set to support dynamic shapes, but there isn't any code to support them.